package.json

```diff
@@ -126,7 +126,7 @@
     "extensions": {
       "codegen": {
         "generates": {
-          "src/terrasoApi/gql/": {
+          "src/terrasoApi/shared/graphqlSchema/": {
```

Comment:
yes, as long as there are fragments and queries defined in the web client codegen will need to happen in the web client. i'm not entirely sure what it will look like as i haven't set that up before, but this config will definitely need to change a bit when that happens